packages/react-components/src/templates/UserProfileHeader.tsx

```diff
@@ -340,13 +340,16 @@ const UserProfileHeader: React.FC<UserProfileHeaderProps> = ({
             <TabLink href={tabRoutes.about({}).$}>Background</TabLink>
             <TabLink href={tabRoutes.outputs({}).$}>
               Shared Outputs
-              {` (${sharedOutputsCount})`}
+              {sharedOutputsCount !== undefined && ` (${sharedOutputsCount})`}
```

Comment:
for these tab count display, might be worth setting a default value in props instead?